xbmc/addons/AddonDll.h

```diff
@@ -43,8 +43,9 @@ namespace ADDON
   class CAddonDll : public CAddon, public ANNOUNCEMENT::IAnnouncer
   {
   public:
-    CAddonDll(const AddonProps &props);
-    CAddonDll(const cp_extension_t *ext);
+    CAddonDll(AddonProps props);
+
+    //FIXME: wrong
```

Comment:
That's probably enough for you to know why, but nobody else will be able to get this I guess?